frontend/packages/component-library-styles/colors.scss

```diff
@@ -1,4 +1,4 @@
-@import 'primitiveColors.css';
+@use 'primitiveColors.scss';
```

Comment:
This comment is out of scope of this ticket and worth investigating later:
I wonder if we just need to replace `@import` with `@use`.
ref: https://medium.com/@philip.mutua/difference-between-use-and-import-in-scss-1cb6f501e649

Reply:
Though this might be the reason why we didn't do that from the start:
https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/componentLibrary/common/README.md?plain=1#L8

Reply:
Created a separate ticket for this for later: https://codedotorg.atlassian.net/browse/DESIGN2-267